### DISCUSSION TITLE
Why can't I run a query on a limit greater than 5000 (intel ioc) ?

### DISCUSSION THREAD

**Discussion by NSH531** [CrowdStrike/falconpy#624] [Q&A] [answered]
To circumvent this limitation I have to run a script that runs at a time on 5000 indicators.

**Comment by jshcodes** [comment]
Hi @NSH531!

This limitation is maintained by the API endpoint, and not FalconPy. 

You're correct that you will need to loop and process indicators in batches due to this limitation. 

We will have an example posted online shortly that specifically demonstrates how to iterate Intel indicators.  Expect this by the end of the month. :smile:

**Reply by NSH531** [reply]
Hi @jshcodes!
I'll glad to get the code sample.
 I'm working on a pretty serious project in the field.
:)

**Reply by jshcodes** [reply]
You got it!  I'll post an update here once that repository moves to public. (Should be happening within the next day or so.)

**Comment by NSH531** [comment]
Thanks

**Comment by jshcodes** [comment, marked as answer]
Hi @NSH531 -

The repo I mentioned is online now and available here: https://github.com/CrowdStrike/MISP-tools

This should give you a good example of pagination. :smile:

**Comment by NSH531** [comment]
In which .py file is the relevant code?

**Reply by jshcodes** [reply]
https://github.com/CrowdStrike/MISP-tools/blob/main/cs_misp_import/intel_client.py#L56
